stix_shifter_modules/guardium/configuration/config.json

```diff
@@ -23,24 +42,36 @@
             "type": "password",
             "optional": true
         },
-        "client_id": {
-            "type": "text",
-            "previous": "connection.options.client_id"
-        },
-        "client_secret": {
-            "type": "password",
-            "previous": "connection.options.client_secret"
-        }
+        "options": {
+            "client_id": {
+                "type": "string",
+                "optional": true,
+                "hidden": true
+            },
+            "client_secret": {
+                "type": "password",
+                "optional": true,
+                "hidden": true
+            },
+            "grant_type": {
+                "type": "string",
+                "optional": true,
+                "hidden": true
+            }
+        } 
     },
     "configuration": {
         "auth": {
-            "type" : "fields",
             "username": {
                 "type": "password"
             },
             "password": {
                 "type": "password"
             }
-        }
+          },
+          "logLevel": {
+            "type": "text",
+             "default": "debug"
```

Comment:
is `logLevel` user configurable and changes based on user account?

Reply:
He can change it if needed.

Reply:
> He can change it if needed.

where does this `logLevel` fields basically used? if it is based on user configuration that it can stay inside configuration object otherwise we need to move it to connection object.

Reply:
done